airbyte-integrations/connectors/source-project-sync/source_project_sync/streams.py

```diff
@@ -0,0 +1,186 @@
+         if you do not need to implement incremental sync for any streams, remove this class.
+    """
+
+    # TODO: Fill in to checkpoint stream reads after N records. This prevents re-reading of data if the stream fails for any reason.
+    state_checkpoint_interval = None
+
+    @property
+    def cursor_field(self) -> str:
+        """
+        TODO
+        Override to return the cursor field used by this stream e.g: an API entity might always use created_at as the cursor field. This is
+        usually id or date based. This field's presence tells the framework this in an incremental stream. Required for incremental.
+        :return str: The name of the cursor field.
+        """
+        return []
+
+    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
+        """
+        Override to determine the latest state after reading the latest record. This typically compared the cursor_field from the latest record and
+        the current state and picks the 'most' recent cursor. This is how a stream's state is determined. Required for incremental.
+        """
+        return {}
+
+
+class FinanceStream(HttpStream):
+    url_base = "https://www.xledger.net/Flex/"
+
+    primary_key = None
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
+
+    def path(
+        self,
+        stream_state: Mapping[str, Any] = None,
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return "112706017670337.json?t=APLF6qr6AIDformleE2ylIO4JVobFhEOOgwYrhOsczHeskNkyONT6VkddFwDUnNtkB6aPnxsSjB_Nxf5mh5vfdvyQ2P6tp2jzobDJTFkWVYR__NFcHG8I1eeKjKiTiVg9wed3_licUXTOAAskDomtJ10XVsZ4Y_wGgPGtdsh5HMltyLaBPtoqiq0soyOWhQ4WFOTCNjPb2MV_qRrEF7waksPdydnNxiP4gDYWHFTnjwTYrHg7L_YDJy5BtvfJpqRg6_G8SE38F1GXG5LxZZVcZXhBqyqorDssO77b02pYr202OZWW_gB7ZsuDyLAYgC20wNdwJL2NZrGTUKLs18v"
+
+    def parse_response(
+        self,
+        response: requests.Response,
+        stream_state: Mapping[str, Any],
+        stream_slice: Mapping[str, Any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> Iterable[Mapping]:
+        for row in response.json().get('rows'):
+            yield {
+                'Prosjekt #': row[0],
+                'Prosjektnavn': row[1],
+                'Hovedprosjekt': row[2],
+                'Prosjekt-ID': row[3],
+                'Firma/Person': row[4],
+                'Prosjektgruppe': row[5],
+                'Prosjektleder': row[6],
+                'Fra': row[7],
+                'Til': row[8],
+                'Kontraktsbeløp': row[9],
+                'Kostnad totalt': row[10],
+                'Tot. est. timer': row[11],
+                'Estimert dekningsgrad': row[12],
+                'Koststed': row[13]
+            }
```

Comment:
this can be generic and moved to the superclass

Reply:
Fixed it!